client/signup/steps/site-picker/site-picker-submit.jsx

```diff
@@ -14,22 +14,20 @@ export class SitePickerSubmit extends Component {
 		const hasPaidPlan = siteHasPaidPlan( selectedSite );
 		const { ID: siteId, slug: siteSlug } = selectedSite;
 
-		this.props.submitSignupStep( {
-			stepName,
-			stepSectionName,
-			siteId,
-			siteSlug,
-		} );
-
 		this.props.submitSignupStep(
-			{ stepName: 'themes', wasSkipped: true },
+			{
+				stepName,
+				stepSectionName,
+				siteId,
+				siteSlug,
+			},
 			{ themeSlugWithRepo: 'pub/twentysixteen' }
 		);
 
 		if ( hasPaidPlan ) {
 			this.props.submitSignupStep(
 				{ stepName: 'plans-site-selected', wasSkipped: true },
-				{ cartItems: null }
+				{ cartItem: null, themeSlugWithRepo: 'pub/twentysixteen' }
```

Comment:
Wondering if we could explain why we switched this from `cartItems` to `cartItem`? I made the changes a little while ago to accommodate the multiple cartItems a plans page might be submitting simultaneously ( add-ons + plans )

https://github.com/Automattic/wp-calypso/pull/81340/files#diff-4c9407fbe3653d84b6fc1dffdf7c75062e99fa46eb9fdcc62d0b544da8d59eaaR32

Reply:
> Wondering if we could explain why we switched this from cartItems to cartItem? 
Good catch 💯 
That was unintended!